contracts/src/v0.1/RequestResponseCoordinator.sol

```diff
@@ -108,6 +105,43 @@ contract RequestResponseCoordinator is
     event MinBalanceSet(uint256 minBalance);
     event PrepaymentSet(address prepayment);
 
+    event DataRequestFulfilledUint256(
```

Comment:
We will have to think how to change the payment because each data type request will have theoretically different price. The easiest would be to make the price high enough so that it covers the most expensive data type. What do you think?